portfilo-freecodecamp-project/portfilo.css

```diff
@@ -0,0 +1,183 @@
+body{
+  font-family: 'Poppins', sans-serif;
+}
+.nav {
+  font-size: 22px;
+  top: 0;
+  display: flex;
+  justify-content: flex-end;
+  position: fixed;
+  width:100%;
+  background-color:#be3144;
+}
+.nav ul {
+  display: flex;
+  margin-right: 2rem;
+}
+.nav-list a:hover {
+  background: #45567d;
+}
+li {
+  list-style: none;
+}
+a{
+  padding: 20px;
+  color: #f0f0f0;
+  text-decoration: none;
+
+}
+
+#welcome-section{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  width: 100%;
+  height: 100vh;
+  background-color: #000;}
+h1  {
+  font-family: 'Raleway', sans-serif;
+  margin-bottom: 2px;
+  font-size: 5rem;
+  text-align: center;
+  color: white;
+}
+h2  {
+  top: 0;
+  font-family: 'Raleway', sans-serif;
+  margin-bottom: 2px;
+  font-size: 42px;
+  text-align: center;
+  color: white;
+  border-bottom: 0.2rem solid white;
+
+}
+
+.welcome-section p {
+  margin-top: 0px;
+  font-size: 3rem;
+  text-align: center;
+  color: #be3144;
+}
+p {
+  align-items: center;
+  margin-top: 0px;
+  font-size: 3rem;
+  text-align: center;
+}
```

Comment:
Always write more specific css, when you are trying to style a particular element in screen, rather than `p`, otherwise, you will have overwrite everywhere else you have used `p` tag.